assignments/objects.js

```diff
@@ -10,35 +10,66 @@
 // 5,adaine5@samsung.com,Antonietta,F
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
-const example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
+const Mitzi = {
+  "id": 1,
+  "name": "Mitzi",
+  "email": "mmelloy0@psu.edu",
   "gender": "F"
 }
 
 // Write your intern objects here:
+const Kennan = {
+  "id": 2,
+  "name": "kennan",
+  "email": "kdiben1@tinypic.com",
+  "gender": "M",
+  "speak":  () => {
+    return ( 'Hello, my name is Kennan')
```

Comment:
This is an interesting way but you can also just console log it for the next challenge